jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -605,6 +606,58 @@ def vue_set_value(self, data):
         value = data.get('value')
         setattr(self, attr_name, value)
 
+    def apply_RGB_presets(self):
+        """
+        Applies preset colors, opacities, and stretch settings to all visible layers
+        (in all viewers) when in Monochromatic mode.
+        """
+
+        if (self.image_color_mode_value != "One color per layer" or
+                self.image_color_mode_sync['mixed']):
+            raise ValueError("RGB presets can only be applied if color mode is Monochromatic.")
+        # Preselected colors we want to use for 5 or less layers
+        preset_colors = [self.swatches_palette[0][0],
+                         "#0000FF",
+                         "#00FF00",
+                         self.swatches_palette[1][0],
+                         self.swatches_palette[4][1]]

```

Comment:
this results in RBG (instead of RGB) for three layers 🤔 ... I think we probably want to apply in chromatic order, but also not just take the first N from this list (for two layers, we probably want RB not RG).

Reply:
Well, this does give RB for two, and for the case of three layers...given that we have no idea if the layers themselves were loaded in chromatic order based on their filters, do we really care if the first three are RBG vs RGB? I would be fine adding more logic, I just don't know if it actually matters.

Reply:
We can ask... eventually we want to allow layer re-ordering, and I think that ordered by wavelength is the most sensible assumption if we're going to make any 🤷

Reply:
I think this is my only remaining hold-up.  We could just have `if n_visible == 2: preset_colors = preset_colors[::4]; elif n_visible == 3: preset_colors = preset_colors[::2]` (and 1, 4, and 5 should be safe to use the full list, unless for 4 we want to skip something in the middle to keep the last entry), but that feels a bit hacky and heavy-handed.